### ISSUE TITLE
Can we run xcuitests in parallel on multiple simulators in a single machine using fastlane ?

### ISSUE THREAD

**Issue by gop1334** [fastlane/fastlane#1055]

**Comment by vpolouchkine** [comment]
It looks like there hasn’t been any recent activity on this issue in some time, so I’m closing it. If this is still a problem, feel free to re-open a new issue and I’d be happy to help :+1:
